axon/threads_test.go

```diff
@@ -157,66 +148,6 @@ func TestDeterministicSingleThreadedTraining(t *testing.T) {
 	assert.False(t, netA.WtsHash() == netB.WtsHash())
 }
 
-func TestMultithreadedSendSpike(t *testing.T) {
```

Comment:
Why delete these? These are great unit tests ensuring the thread-safety of big parts of the cycle-level processing.

Reply:
@rcoreilly Can I bring these back?

Reply:
There is no separate parameterization for sendspike vs. anything else here -- all threading using single NThreads param.  Everything is using neuron level parallel and I didn't see any speed diffs for the diff functions, and the complexity cost is significant, so much simpler to use one NThreads.